editor/src/components/canvas/commands/queue-group-true-up-command.ts

```diff
@@ -12,14 +12,14 @@ import { trueUpTargetToDescription } from '../../../core/model/groups'
 
 export interface QueueGroupTrueUp extends BaseCommand {
   type: 'QUEUE_GROUP_TRUE_UP'
-  target: TrueUpTarget
+  targets: Array<TrueUpTarget>
```

Comment:
Since we have arrays of commands, is there a distinction between one `QueueGroupTrueUp` with X `targets` and X instances of `QueueGroupTrueUp` with a target in each?

Reply:
I don't think so! maybe some ultra minor performance difference? it's a single editor state patch instead of `n` patches folded